### PR TITLE
Updates to CrosscheckFingerprints documentation.

### PR DESCRIPTION
**Pull request by samuelklee**
Changes comments on the fields in the CrosscheckMetric class to Javadoc, in response to https://gatk.broadinstitute.org/hc/en-us/community/posts/360058443032-Interpretting-CrosscheckFingerprints-metrics and subsequent discussion on Slack.

Reviewers can use this PR to add additional documentation and updates, if desired.

----

### Checklist (never delete this)

Never delete this, it is our record that procedure was followed. If you find that for whatever reason one of the checklist points doesn't apply to your PR, you can leave it unchecked but please add an explanation below.

#### Content
- [x] Added or modified tests to cover changes and any new functionality
- [x] Edited the README / documentation (if applicable)
- [x] All tests passing on Travis

#### Review
- [x] Final thumbs-up from reviewer
- [x] Rebase, squash and reword as applicable

For more detailed guidelines, see https://github.com/broadinstitute/picard/wiki/Guidelines-for-pull-requests

### CODE REVIEW

File: src/main/java/picard/fingerprint/CrosscheckFingerprints.java
Comment (samuelklee) [inline src/main/java/picard/fingerprint/CrosscheckFingerprints.java:81]: We may want to expand upon this documentation, I just fixed up a few typos I noticed. I also found it confusing that this text was mostly---but not faithfully---reproduced in the tool summary (and hence did not fix the corresponding typos there).

File: src/main/java/picard/fingerprint/CrosscheckMetric.java
Comment (samuelklee) [inline src/main/java/picard/fingerprint/CrosscheckMetric.java:62]: These descriptions should probably be expanded, I just changed them to Javadoc and fixed up a few LEFT vs. RIGHT typos.

File: src/main/java/picard/fingerprint/CrosscheckFingerprints.java
Comment (samuelklee) [reply src/main/java/picard/fingerprint/CrosscheckFingerprints.java:81]: One other comment while I'm here---"moltenize/d" is a very idiosyncratic term to describe this type of table reshaping. Why not just "molten/melted"?

Comment (yfarjoun) [reply src/main/java/picard/fingerprint/CrosscheckFingerprints.java:81]: I have no strong feelings about moltenized....if you think that "molten" is more clear, go for it!

**Comment by samuelklee** [comment]
Thanks @yfarjoun! As discussed on Slack with Comms, they’ll take over this PR and use it to address the above comments and make further changes.

**Comment by derekca** [comment]
Hi  @yfarjoun, I've made a few clarifying revisions to the intro — [CrosscheckFingerprints_Revised](https://github.com/broadinstitute/picard/files/4990547/CrosscheckFingerprints_Revised.txt)

Please take a look and incorporate them as appropriate.

**Comment by yfarjoun** [comment]
why are these changes not integrated into the PR? I'm confused about how
you want to work here…..

**Comment by kbergin** [comment]
Hi @derekca - would you be able to incorporate your suggested wording into this PR? Going through open Picard PRs and hoping to get them merged in :)

**Comment by derekca** [comment]
@kbergin Just found out about this now... I never got pinged for these responses for some reason.

I either do not have the correct permissions, or am not familiar enough with git, to be able to incorporate these changes to the doc.

**Comment by kbergin** [comment]
@samuelklee would you be able to incorporate the changes that Derek linked above and close out this PR?

**Comment by gbggrant** [comment]
Hi @samuelklee I'm assigning this back to you as it seems to have stalled out.

**Comment by samuelklee** [comment]
Apologies @gbggrant, I should've noted here that User Ed is on this, as was decided on the gatk-userliaison Slack.

I'm probably not actually familiar enough with the tool to make detailed changes or provide a review---I originally opened the PR to help get the ball rolling in response to some discussions I happened to see on Slack and the GATK Forum. I made some obvious changes and fixed some typos, but I think @derekca or someone else more qualified than me will have to bring this home!

**Comment by jessicaway** [comment]
@knoblett Has the User Ed team has had a chance to look at this?

**Comment by gbggrant** [comment]
@samuelklee is there anything holding this PR up from being merged?  It's been waiting a while.

**Review by jessicaway — APPROVED** [review]

**Comment by samuelklee** [comment]
@gbggrant sorry, just seeing your mention after the long weekend. Just wanted to note that this PR was never completed, to my knowledge. As stated a few times above, I made some cursory fixes to get the ball rolling but handed things over to User Ed to finish up. It looks like @derekca came up with some additional changes in https://github.com/broadinstitute/picard/pull/1520#issuecomment-665219785 that were never incorporated. Perhaps it's worth opening another PR to merge those changes?

**Comment by gbggrant** [comment]
@samuelklee can you and @derekca work to get the updated documentation integrated into this PR?  I have tried reading the text file provided in the comment above, but it is in a very different format than used in the picard comment and usage statement.

**Comment by samuelklee** [comment]
@gbggrant thanks for making sure this doesn’t fall through the cracks! Unfortunately, my position is still the same as in https://github.com/broadinstitute/picard/pull/1520#issuecomment-801559995 (and I’m also currently on vacation!) User Ed, somebody more familiar with the tool, or anyone else with the bandwidth to review and incorporate the changes in the text file should bring this home.

Again, I only opened this PR as a courtesy with the understanding that others would close it out—no good deed, I suppose! If no one else cares to finish it up, then feel free to close it. Seems a bit of shame to miss a chance to improve documentation, but it’s no skin off my back.

**Comment by samuelklee** [comment]
@bhanugandham could I possibly recruit you to reassign or otherwise shepherd this PR?

I'm assuming the edits suggested above by @derekca are good---the only nitpick I have upon skimming is that I still think we should still use the standard "molten" to refer to the dataframe melt operation, instead of the rather idiosyncratic "moltenized" (which has <500 hits on Google!)

Can you find someone to incorporate those edits along with the changes I've made to the code? I think you should just be able to add onto this PR, but feel free to copy the branch and open another if needed.

I'm fine with punting to additional changes to the Javadoc, although as commented above I think they could bear expansion.

**Comment by derekca** [comment]
> I'm assuming the edits suggested above by @derekca are good and are an improvement over any typo fixes I might've introduced above. The only nitpick I have upon skimming is that I still think we should still use the standard "molten" to refer to the dataframe melt operation, instead of the rather idiosyncratic "moltenized" (which has <500 hits on Google!)

Thanks for the input, I've fixed the comment about molten outputs in the new version.

I've gone ahead and created a separate pull request, [1721](https://github.com/broadinstitute/picard/pull/1721), for revising the introduction to this doc.

**Comment by samuelklee** [comment]
Great, thanks @derekca! @gbggrant I've rebased and will wait for tests to pass again before merging. Perhaps then @derekca can resolve any conflicts over in his PR and merge afterwards.

**Comment by bhanugandham** [comment]
@samuelklee and @gbggrant I apologize i didn't see the comments. I was on vacation the last week and a half and just got back today. Looks like @derekca has taken care of this? Please let me know if there is anything else i can help with. Sorry again for the delay!

**Comment by samuelklee** [comment]
No worries, @bhanugandham. Sorry, didn’t realize you were off! I think @derekca is on top of the other PR. Not sure who will review, but I’ll leave that up to you all. Thanks again to both of you!

**Comment by bhanugandham** [comment]
Sounds good! Thanks Sam!

**Pull request closed without merging**
